messages/format.js

```diff
@@ -0,0 +1,24 @@
+const fs = require('fs');
+const path = require('path');
+const file = require('../helper/file.js');
+
+module.exports =
+{
+    format: function (fileName, ...replacements)
+    {
+        var filePath = path.join(__dirname, "templates", fileName);
```

Comment:
Change var to const if it's a constant and let if it will be re-assigned please

Reply:
const and let statements used